core/src/main/java/org/apache/calcite/config/CalciteConnectionProperty.java

```diff
@@ -159,7 +159,10 @@
 
   /** Whether to make create implicit functions if functions do not exist
    * in the operator table, default false. */
-  LENIENT_OPERATOR_LOOKUP("lenientOperatorLookup", Type.BOOLEAN, false, false);
+  LENIENT_OPERATOR_LOOKUP("lenientOperatorLookup", Type.BOOLEAN, false, false),
+
+  /** Whether to enable top-down optimization in Volcano planner. */
+  VOLCANO_TOPDOWN_OPT("enableVolcanoTopDownOptimization", Type.BOOLEAN, false, false);
```

Comment:
How about remove `VOLCANO_`? volcano should be kept as inside term only.

Reply:
Removed `VOLCANO_`